pkg/cephmgr/osd/scheme.go

```diff
@@ -26,21 +26,29 @@ import (
 )
 
 const (
-	WalPartitionName      = "wal"
-	DatabasePartitionName = "db"
-	BlockPartitionName    = "block"
-	walPartition          = 1
-	databasePartition     = 2
-	blockPartition        = 3
-	UseRemainingSpace     = -1
-	schemeFilename        = "partition-scheme"
-	WalDefaultSizeMB      = 576
-	DBDefaultSizeMB       = 20480
+	Filestore                     = "filestore"
```

Comment:
Im ok if you dont make them enum in this PR but it would good practice to use them. It would keep the code more organized and the Constant variable grouping more obvious :)